Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM python:3
+FROM python:3-slim
 
-# Install NodeJS to the Alpine container (depedency).
+# Install g++ and make to ensure dependencies install with python:3-slim.
 RUN apt-get -y update
+RUN apt-get -y install g++ make
```

Comment:
This is needed for some Python dependencies to properly install on `python:3-slim`.